src/tests/test_raft_repl_dev.cpp

```diff
@@ -548,13 +558,24 @@ TEST_F(RaftReplDevTest, Drop_Raft_Entry_Switch_Leader) {
 }
 #endif
 
-// TODO
-// double restart:
-// 1. restart one follower(F1) while I/O keep running.
-// 2. after F1 reboots and leader is resyncing with F1 (after sending the appended entries), this leader also retarts.
-// 3. F1 should receive error from grpc saying originator not there.
-// 4. F2 should be appending entries to F1 and F1 should be able to catch up with F2 (fetch data from F2).
 //
+// This test case should be run in long running mode to see the effect of snapshot and compaction
+// Example:
+// ./bin/test_raft_repl_dev --gtest_filter=*Snapshot_and_Compact* --log_mods replication:debug --num_io=999999
+// --snapshot_distance=200 --num_raft_logs_resv=20000 --res_mgr_audit_timer_ms=120000
+//
+TEST_F(RaftReplDevTest, Snapshot_and_Compact) {
```

Comment:
It would be better to verify that a snapshot and compact was called. May be in a future PR.